Added all upgrade docs (#4925)
umbraco/UmbracoDocs: 10/umbraco-cms/fundamentals/setup/upgrading/version-specific/minor-upgrades-for-umbraco-8.md

---

> description: This articles provides details on how to upgrade to the next minor version when using Umbraco 8.

Reviewer: ```suggestion
description: This article provides details on how to upgrade to the next minor version when using Umbraco 8.
```

---

# Minor upgrades for Umbraco 8

Sometimes there are exceptions to these guidelines, which are listed in the **[version-specific guide](README.md)**.

## Note

> It is necessary to run the upgrade installer on each environment of your Umbraco site. So if you want to update your staging and your live site then you need to repeat the steps below and make sure that you click through the install screens so that your upgrade is complete.

Reviewer 2: ```suggestion
It is necessary to run the upgrade installer on each environment of your Umbraco site. If you want to update your staging and live site you need to repeat the steps below. Make sure you click through the install screens so that your upgrade is complete.
```

## Contents

> In this article you will find instruction of 3 different ways of upgrading:

Reviewer: ```suggestion
In this article you will find instructions for 3 different ways of upgrading:
```

The upgrade will run through all the files and make sure you have the latest changes while leaving files you have updated.

## Upgrade manually from a zip file

Download the `.zip` file for the new version you are upgrading to from [https://our.umbraco.com/download](https://our.umbraco.com/download)

Copy the following folders from inside the `.zip` file over the existing folders in your site:

* `/bin`
* `/Umbraco`

{% hint style="info" %}
There are hosting providers (we know of one: RackSpace Cloud) that require proper casing of file and folder names. Normally on Windows this is not a problem. If your hosting provider however forces proper casing, you will need to verify that the folder and file names are in the same casing as in the newest version you're upgrading to.
{% endhint %}

### Merge configuration files

You can expect some changes to the following configuration files:

* Any file in the `/Config` folder
* The `/Global.asax` file
* The `web.config` file in the root of your site **(Important: make sure to copy back the version number, and the connection string as they were.)**
* In rare cases, the `web.config` file in the `/Views` folder

> Use a tool like [WinMerge](http://winmerge.org/ "WinMerge") to check changes between all of the config files. Depending on when you last did this there may have been updates to quite a few of them.

Reviewer: ```suggestion
Use a tool like [WinMerge](http://winmerge.org/ "WinMerge") to check changes between all of the config files. Depending on when you last did this there may have been updates to few of them.
```

> There's also the possibility that some files in the `/Config` folder are new or some have been removed (we do make a note of this in the release notes). WinMerge (and other diff tools) is able to compare folders as well so you can spot these differences.

Reviewer: ```suggestion
There's also the possibility that some files in the `/Config` folder are new or some have been removed (we do make a note of this in the release notes). WinMerge (and other diff tools) can compare folders as well so you can spot these differences.
```
